runtime-sdk/modules/contracts/src/abi/oasis/mod.rs

```diff
@@ -158,7 +154,11 @@ impl<Cfg: Config> OasisV1<Cfg> {
                 // Compute how much gas was wanted.
                 let final_gas = gas::get_remaining_gas(instance);
                 let wanted_gas = initial_gas + exhausted_gas.saturating_sub(final_gas);
-                core::Error::OutOfGas(initial_gas, wanted_gas).into()
+                core::Error::out_of_gas::<<<C::Runtime as Runtime>::Core as core::API>::Config>(
```

Comment:
:dizzy_face:

Reply:
Yeah unfortunately the Rust compiler wants explicit casts here :(